stack/build.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:jammy
+
+ARG sources
+ARG packages
+
+RUN echo "$sources" > /etc/apt/sources.list
+
+RUN echo "debconf debconf/frontend select noninteractive" | debconf-set-selections && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get -y $package_args update && \
+  apt-get -y $package_args upgrade && \
+  apt-get -y $package_args install locales && \
+  locale-gen en_US.UTF-8 && \
+  update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 && \
+  apt-get -y $package_args install $packages && \
+  rm -rf /var/lib/apt/lists/* /tmp/*
+
+RUN for path in /workspace /workspace/source-ws /workspace/source; do git config --system --add safe.directory "${path}"; done
+
+RUN curl -sL -o /usr/local/bin/yj https://github.com/sclevine/yj/releases/latest/download/yj-linux \
```

Comment:
This URL is wrong - it should be `yj-linux-amd64`. We should also ensure that `curl` fails on error. Let's use the following command:

```
RUN curl -sSfL -o /usr/local/bin/yj https://github.com/sclevine/yj/releases/latest/download/yj-linux-amd64 \
  && chmod +x /usr/local/bin/yj
```

Reply:
(from :https://github.com/paketo-buildpacks/stacks/blob/c00defb718858eb3724c71addae362c9c047f068/bionic/dockerfile/build/Dockerfile#L21)

Reply:
This was copied from https://github.com/paketo-buildpacks/jammy-tiny-stack/blob/e9a41436b06d3c3fa2960980ea98be36796ab3b6/stack/build.Dockerfile#L20-L21

I will create a PR to fix it also there

Reply:
Perfect, thank you. Yes, we copied that command before we fixed it in `bionic` and clearly forgot to port the fix over to `jammy`. Thanks for submitting a PR there, too.

Reply:
https://github.com/paketo-buildpacks/jammy-tiny-stack/pull/10